src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -13,44 +13,44 @@ using namespace winrt::TerminalApp;
-static constexpr std::wstring_view SWITCHTOTAB7_KEY{ L"switchToTab7" };
-static constexpr std::wstring_view SWITCHTOTAB8_KEY{ L"switchToTab8" };
-static constexpr std::wstring_view OPENSETTINGS_KEY{ L"openSettings" };
+static constexpr std::string_view COPYTEXT_KEY{ "copy" };
+static constexpr std::string_view PASTETEXT_KEY{ "paste" };
+static constexpr std::string_view NEWTAB_KEY{ "newTab" };
+static constexpr std::string_view NEWTABWITHPROFILE0_KEY{ "newTabProfile0" };
+static constexpr std::string_view NEWTABWITHPROFILE1_KEY{ "newTabProfile1" };
+static constexpr std::string_view NEWTABWITHPROFILE2_KEY{ "newTabProfile2" };
+static constexpr std::string_view NEWTABWITHPROFILE3_KEY{ "newTabProfile3" };
+static constexpr std::string_view NEWTABWITHPROFILE4_KEY{ "newTabProfile4" };
+static constexpr std::string_view NEWTABWITHPROFILE5_KEY{ "newTabProfile5" };
+static constexpr std::string_view NEWTABWITHPROFILE6_KEY{ "newTabProfile6" };
+static constexpr std::string_view NEWTABWITHPROFILE7_KEY{ "newTabProfile7" };
+static constexpr std::string_view NEWTABWITHPROFILE8_KEY{ "newTabProfile8" };
+static constexpr std::string_view NEWWINDOW_KEY{ "newWindow" };
+static constexpr std::string_view CLOSEWINDOW_KEY{ "closeWindow" };
+static constexpr std::string_view CLOSETAB_KEY{ "closeTab" };
+static constexpr std::string_view SWITCHTOTAB_KEY{ "switchToTab" };
+static constexpr std::string_view NEXTTAB_KEY{ "nextTab" };
+static constexpr std::string_view PREVTAB_KEY{ "prevTab" };
+static constexpr std::string_view INCREASEFONTSIZE_KEY{ "increaseFontSize" };
+static constexpr std::string_view DECREASEFONTSIZE_KEY{ "decreaseFontSize" };
+static constexpr std::string_view SCROLLUP_KEY{ "scrollUp" };
+static constexpr std::string_view SCROLLDOWN_KEY{ "scrollDown" };
+static constexpr std::string_view SCROLLUPPAGE_KEY{ "scrollUpPage" };
+static constexpr std::string_view SCROLLDOWNPAGE_KEY{ "scrollDownPage" };
+static constexpr std::string_view SWITCHTOTAB0_KEY{ "switchToTab0" };
+static constexpr std::string_view SWITCHTOTAB1_KEY{ "switchToTab1" };
+static constexpr std::string_view SWITCHTOTAB2_KEY{ "switchToTab2" };
+static constexpr std::string_view SWITCHTOTAB3_KEY{ "switchToTab3" };
+static constexpr std::string_view SWITCHTOTAB4_KEY{ "switchToTab4" };
+static constexpr std::string_view SWITCHTOTAB5_KEY{ "switchToTab5" };
+static constexpr std::string_view SWITCHTOTAB6_KEY{ "switchToTab6" };
+static constexpr std::string_view SWITCHTOTAB7_KEY{ "switchToTab7" };
+static constexpr std::string_view SWITCHTOTAB8_KEY{ "switchToTab8" };
+static constexpr std::string_view OPENSETTINGS_KEY{ "openSettings" };
 
 // Specifically use a map here over an unordered_map. We want to be able to
 // iterate over these entries in-order when we're serializing the keybindings.
-static const std::map<std::wstring_view, ShortcutAction> commandNames {
+static const std::map<std::string_view, ShortcutAction> commandNames {

```

Comment:
alas, this one should probably be a `map<string,ShortcutAction,std::less<>>` -- we need to use string for map keys because string_view is just a pointer+length/technically a temporary. also adding `less<>` gives us #972 !

Reply:
Chatting with @adiviness: this is "fine" here, but can catch out an unsuspecting person if the map ever becomes non-`const`. If we wholeheartedly believe this map will never become non-const _assuming proper design_, let's leave it as `string_view` and comment it heavily.

Reply:
it should be ok in this circumstance because the strings are all not moving. maybe a comment to explain why it's ok here specifically?